src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -61,8 +59,10 @@ public New3Command(string commandName, ITemplateEngineHost host, ITelemetryLogge
             CommandName = commandName;
```

Comment:
Looks like the `host = _host = new ExtendedTemplateEngineHost(host, this);` no longer requires the `_host =` portion either